src/main/java/gregtech/integration/theoneprobe/provider/QuantumEnergyProvider.java

```diff
@@ -0,0 +1,42 @@
+package gregtech.integration.theoneprobe.provider;
+
+import gregtech.api.GTValues;
+import gregtech.api.capability.IQuantumController;
+import gregtech.api.metatileentity.interfaces.IGregTechTileEntity;
+import gregtech.api.util.GTUtility;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraft.world.World;
+
+import mcjty.theoneprobe.api.IProbeHitData;
+import mcjty.theoneprobe.api.IProbeInfo;
+import mcjty.theoneprobe.api.IProbeInfoProvider;
+import mcjty.theoneprobe.api.ProbeMode;
+import mcjty.theoneprobe.api.TextStyleClass;
+
+public class QuantumEnergyProvider implements IProbeInfoProvider {
+
+    @Override
+    public String getID() {
+        return ":quantum_energy_provider";
+    }
+
+    @Override
+    public void addProbeInfo(ProbeMode mode, IProbeInfo probeInfo, EntityPlayer player, World world,
+                             IBlockState blockState, IProbeHitData data) {
+        if (blockState.getBlock().hasTileEntity(blockState) &&
+                world.getTileEntity(data.getPos()) instanceof IGregTechTileEntity gtte) {
+            if (gtte.getMetaTileEntity() instanceof IQuantumController controller) {
+                configureEnergyUsage(controller.getEnergyUsage() / 10, probeInfo);
```

Comment:
Why is the energy usage here divided by 10, when the quantum controller just removes all of `getEnergyUsage` twice per second?

Reply:
this displays the EU per tick, but the energy usage is per 10 ticks, so it's divided by 10